Content.Shared/Chemistry/Reagent/ReagentEffect.cs

```diff
@@ -82,6 +82,7 @@ public enum ReactionMethod
         ReagentPrototype Reagent,
         FixedPoint2 Quantity,
         IEntityManager EntityManager,
-        ReactionMethod? Method
+        ReactionMethod? Method,
+        ReagentEffectsEntry? MetabolismEffects
```

Comment:
if this is being used as 'metabolism and not touch/etc' it should just be a bool for that since i dont see this data actually being used. furthermore shouldnt it apply regardless?